### PR TITLE
Attribute refactor

### PR DESCRIPTION
**Pull request by sorongisme**
Change wishlist_id, wishlist_name to id, name
Change item_id to id.

### CODE REVIEW

File: service/models.py [hunk 1]
```diff
@@ -33,43 +33,57 @@ class Wishlist(db.Model):
     app = None
 
     # Table Schema
-    wishlist_id = db.Column(db.Integer, primary_key=True)
-    wishlist_name = db.Column(db.String(63), nullable=False)
+    
+    #wishlist_id = db.Column(db.Integer, primary_key=True)
+    #wishlist_name = db.Column(db.String(63), nullable=False)
```
Comment (sakshamgarg) [inline service/models.py:38]: You can remove this comment.

File: service/models.py [hunk 2]
```diff
@@ -33,43 +33,57 @@ class Wishlist(db.Model):
...
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(63), nullable=False)
     owner_id = db.Column(db.Integer, nullable=False)
     created_at = db.Column(db.DateTime, default=datetime.datetime.now())
     wishlist_items = db.relationship("Item", backref="wishlist", cascade="all, delete", lazy=True)
 
     def __repr__(self):
-        return f"<Wishlist {self.wishlist_name} id=[{self.wishlist_id}]>"
-
+        #return f"<Wishlist {self.wishlist_name} id=[{self.wishlist_id}]>"
```
Comment (sakshamgarg) [inline service/models.py:47]: You can remove this comment.

File: service/models.py [hunk 3]
```diff
@@ -33,43 +33,57 @@ class Wishlist(db.Model):
...
+        return f"<Wishlist {self.name} id=[{self.id}]>"
+    
     def create(self):
         """
         Creates a Wishlist to the database
         """
-        logger.info("Creating %s", self.wishlist_name)
-        self.wishlist_id = None  # pylint: disable=invalid-name
+        #logger.info("Creating %s", self.wishlist_name)
```
Comment (sakshamgarg) [inline service/models.py:54]: You can remove this comment.

File: service/models.py [hunk 4]
```diff
@@ -33,43 +33,57 @@ class Wishlist(db.Model):
...
+        logger.info("Creating %s", self.name)
+        #self.wishlist_id = None  # pylint: disable=invalid-name
```
Comment (sakshamgarg) [inline service/models.py:56]: You can remove this comment.

File: service/models.py [hunk 5]
```diff
@@ -145,10 +162,15 @@ def find_by_owner_id(cls, owner_id):
         return cls.query.filter(cls.owner_id == owner_id)
 
     @classmethod
-    def find_or_404(cls, wishlist_id):
+    #def find_or_404(cls, wishlist_id):
+    #    """ Finds a wishlist item by it's ID """
+    #    logger.info("Processing lookup or 404 for id %s ...", wishlist_id)
+    #    return cls.query.get_or_404(wishlist_id)
```
Comment (sakshamgarg) [inline service/models.py:168]: You can remove this comment.

File: service/models.py [hunk 6]
```diff
@@ -133,7 +149,8 @@ def find_by_name(cls, name):
             name (string): the name of the Wishlist you want to match
         """
         logger.info("Processing name query for %s ...", name)
-        return cls.query.filter(cls.wishlist_name == name)
+        #return cls.query.filter(cls.wishlist_name == name)
```
Comment (sakshamgarg) [inline service/models.py:152]: You can remove this comment.

File: service/models.py [hunk 7]
```diff
@@ -120,10 +135,11 @@ def all(cls):
         return cls.query.all()
 
     @classmethod
-    def find(cls, wishlist_id):
+    def find(cls, id):
         """ Finds a Wishlist by it's ID """
-        logger.info("Processing lookup for id %s ...", wishlist_id)
-        return cls.query.get(wishlist_id)
+        #logger.info("Processing lookup for id %s ...", wishlist_id)
```
Comment (sakshamgarg) [inline service/models.py:140]: You can remove this comment.

File: service/models.py [hunk 8]
```diff
@@ -81,7 +95,8 @@ def deserialize(self, data):
             data (dict): A dictionary containing the resource data
         """
         try:
-            self.wishlist_name = data["wishlist_name"]
+            #self.wishlist_name = data["wishlist_name"]
```
Comment (sakshamgarg) [inline service/models.py:98]: You can remove this comment.

File: service/models.py [hunk 9]
```diff
@@ -33,43 +33,57 @@ class Wishlist(db.Model):
...
         db.session.commit()
 
     def serialize(self):
         """ Serializes a Wishlist into a dictionary """
-        return {"wishlist_id": self.wishlist_id,
-                "wishlist_name": self.wishlist_name,
+        #return {"wishlist_id": self.wishlist_id,
+        #        "wishlist_name": self.wishlist_name,
+        #        "owner_id": self.owner_id,
+        #        "created_at": self.created_at}
```
Comment (sakshamgarg) [inline service/models.py:84]: You can remove this comment.

File: service/models.py [hunk 10]
```diff
@@ -33,43 +33,57 @@ class Wishlist(db.Model):
...
+        logger.info("Saving %s", self.name)
+        #if not self.wishlist_id:
+        if not self.id:
             raise DataValidationError("Update called with empty ID field")
         db.session.commit()
 
     def delete(self):
         """ Removes a Wishlist from the data store """
-        logger.info("Deleting %s", self.wishlist_name)
+        #logger.info("Deleting %s", self.wishlist_name)
```
Comment (sakshamgarg) [inline service/models.py:74]: You can remove this comment.

File: service/models.py [hunk 11]
```diff
@@ -33,43 +33,57 @@ class Wishlist(db.Model):
...
         db.session.add(self)
         db.session.commit()
 
     def update(self):
         """
         Updates a Wishlist to the database
         """
-        logger.info("Saving %s", self.wishlist_name)
-        if not self.wishlist_id:
+        #logger.info("Saving %s", self.wishlist_name)
```
Comment (sakshamgarg) [inline service/models.py:65]: You can remove this comment.

File: service/models.py [hunk 12]
```diff
@@ -33,43 +33,57 @@ class Wishlist(db.Model):
...
+        logger.info("Saving %s", self.name)
+        #if not self.wishlist_id:
```
Comment (sakshamgarg) [inline service/models.py:67]: You can remove this comment.

File: tests/test_wishlists_model.py [hunk 13]
```diff
@@ -59,11 +59,14 @@ def tearDown(self):
     def test_create_a_wishlist(self):
         """It should Create a wishlist and assert that it exists"""
         create_time = datetime.datetime.now()
-        wishlist = Wishlist(wishlist_name="first wishlist", owner_id=1, created_at=create_time)
+        #wishlist = Wishlist(wishlist_name="first wishlist", owner_id=1, created_at=create_time)
```
Comment (sakshamgarg) [inline tests/test_wishlists_model.py:62]: You can remove this comment.

File: tests/test_wishlists_model.py [hunk 14]
```diff
@@ -59,11 +59,14 @@ def tearDown(self):
...
+        wishlist = Wishlist(name="first wishlist", owner_id=1, created_at=create_time)
         self.assertEqual(str(wishlist), "<Wishlist first wishlist id=[None]>")
         self.assertTrue(wishlist is not None)
-        self.assertEqual(wishlist.wishlist_id, None)
-        self.assertEqual(wishlist.wishlist_name, "first wishlist")
+        #self.assertEqual(wishlist.wishlist_id, None)
+        self.assertEqual(wishlist.id, None)
+        #self.assertEqual(wishlist.wishlist_name, "first wishlist")
```
Comment (sakshamgarg) [inline tests/test_wishlists_model.py:68]: You can remove this comment.

File: tests/test_wishlists_model.py [hunk 15]
```diff
@@ -72,56 +75,59 @@ def test_add_a_wishlist(self):
         wishlists = Wishlist.all()
         self.assertEqual(wishlists, [])
         current_time = datetime.datetime.now()
-        wishlist = Wishlist(wishlist_name="first wishlist", owner_id=1, created_at=current_time)
+        #wishlist = Wishlist(wishlist_name="first wishlist", owner_id=1, created_at=current_time)
```
Comment (sakshamgarg) [inline tests/test_wishlists_model.py:78]: You can remove this comment.

File: tests/test_wishlists_model.py [hunk 16]
```diff
@@ -72,56 +75,59 @@ def test_add_a_wishlist(self):
...
+        wishlist = Wishlist(name="first wishlist", owner_id=1, created_at=current_time)
         self.assertTrue(wishlist is not None)
-        self.assertEqual(wishlist.wishlist_id, None)
+        #self.assertEqual(wishlist.wishlist_id, None)
```
Comment (sakshamgarg) [inline tests/test_wishlists_model.py:81]: You can remove this comment.

File: tests/test_wishlists_model.py [hunk 17]
```diff
@@ -72,56 +75,59 @@ def test_add_a_wishlist(self):
...
+        self.assertEqual(wishlist.id, None)
         wishlist.create()
         # Assert that it was assigned an id and shows up in the database
-        self.assertIsNotNone(wishlist.wishlist_id)
+        #self.assertIsNotNone(wishlist.wishlist_id)
```
Comment (sakshamgarg) [inline tests/test_wishlists_model.py:85]: You can remove this comment.

**Review by sakshamgarg — CHANGES_REQUESTED** [review]

**Review by sakshamgarg — APPROVED** [review]
Looks good to me. Thank you for making the changes.

**Review by reubenabraham — APPROVED** [review]
LGTM

**Pull request merged**